### DISCUSSION TITLE
Trying it for the first time with Svelte 5

### DISCUSSION THREAD

**Discussion by frederikhors** [paoloricciuti/sveltekit-search-params#83] [Q&A]
Hi @paoloricciuti,

this is just a quick example of my issue with your amazing package.

I'm creating a more complete example.

If you open [this reproduction page](https://stackblitz.com/edit/sveltejs-kit-template-default-v1o6ct?file=src%2Froutes%2F%2Bpage.svelte) you can see in the page that the URL is:

```
url: https://sveltejskittemplatedefaultmmkg-3mpx--5173--dc4d7514.local-credentialless.webcontainer.io/?condition=%7B%22or%22%3A%5B%5D%7D&pagination=%7B%22page%22%3A1%2C%22size%22%3A5%7D
```

instead of what I expect:

```
url: https://sveltejskittemplatedefaultmmkg-3mpx--5173--dc4d7514.local-credentialless.webcontainer.io/
```

and I'm also using `{ showDefaults: false }`.

Can I ask you why?

This is a problem because it's like you navigate to new pages many times. _(On stackblitz it's difficult to make it work since if I open the project in a new page it doesn't work, but if you download and try offline you'll see that it takes more clicks to go back instead of one.)_

And I would like 0 click since there are default values ​​at the beginning.

Another question is how to synchronize the forward and backward of the browser with `condition` and `pagination` which is not working now.

If I understand how to do it I can create a PR for docs with Svelte 5.

Can you help me?

**Comment by paoloricciuti** [comment]
Hi I'll take a look as soon as possible 🤟🏻
